package.json

```diff
@@ -99,7 +98,7 @@
     "comment-json": "4.2.5",
     "configstore": "7.0.0",
     "content-type": "1.0.5",
-    "cookie": "0.7.2",
+    "cookie": "^1.0.2",
```

Comment:
not to self: I think I'm supposed to pin this and `open` because of something something yarn lockfile

Reply:
Yeah, we should fix these two to a specific version.